packages/mathtype-froala/README.md

```diff
@@ -8,8 +8,6 @@ Type and handwrite mathematical notation with MathType.
 
 Easily include quality math equations in your documents and digital content.
 
-![MathType for Froala screenshot](http://www.wiris.com/en/system/files/froala_wiris.png)
-

```

Comment:
Even though removing the image is a solution to the broken link error, I believe it would be better to recover the original image.

Please do that.